src/uniparc/components/landing-page/LandingPage.tsx

```diff
@@ -0,0 +1,291 @@
+    groupedDatabases.push(databases.slice(0, 5));
+  } else {
+    let nGroups = 3;
+    if (mediumScreen) {
+      nGroups = 2;
+    }
+    if (smallScreen) {
+      nGroups = 1;
+    }
+    const countByColumn = Math.ceil(databases.length / nGroups);
+    let start = 0;
+    while (start < databases.length) {
+      groupedDatabases.push(databases.slice(start, start + countByColumn));
+      start += countByColumn;
+    }
+  }
+
+  return (
+    <div className={styles['landing-page']}>
+      <HTMLHead title="UniProt sequence archive (UniParc)">
+        <meta name="description" content={metaDescription} />
+      </HTMLHead>
+      <section className="uniprot-grid">
+        <h1 className="uniprot-grid-cell--span-12">UniParc sequence archive</h1>
+        <div
+          className={cn(
+            'uniprot-grid-cell--small-span-12',
+            'uniprot-grid-cell--medium-span-4',
+            styles['image-container']
+          )}
+        >
+          <img src={ArchiveIllustration} width={250} height={250} alt="" />
+        </div>
+        <div className="uniprot-grid-cell--small-span-12 uniprot-grid-cell--medium-span-8">
+          <p>
+            The{' '}
+            <Link to={getLocationEntryPath(Location.HelpEntry, 'uniparc')}>
+              UniProt Archive (UniParc)
+            </Link>{' '}
+            is a comprehensive and non-redundant database that contains most of
+            the publicly available protein sequences in the world. Proteins may
+            exist in different source databases and in multiple copies in the
+            same database. UniParc removes such redundancy by storing each
+            unique sequence only once and giving it a stable and unique
+            identifier (UPI) making it possible to identify the same protein
+            from different source databases. A UPI is never removed, changed or
+            reassigned. UniParc contains only protein sequences and
+            cross-references. All other information about the protein must be
+            retrieved from the source databases using the database
+            cross-references. UniParc tracks sequence changes in the source
+            databases and archives the history of all changes. UniParc has
+            combined many databases into one at the sequence level and searching
+            UniParc is equivalent to searching many databases simultaneously.
+          </p>
+          <p>
+            <Link
+              to={{
+                pathname: LocationToPath[Location.UniParcResults],
+                search: stringifyQuery({ query: '*' }),
+              }}
+            >
+              Start searching in UniParc »
+            </Link>
+          </p>
+        </div>
+
+        {/* Table */}
+        <section
+          className={cn(
+            'uniprot-grid-cell--small-span-12',
+            `uniprot-grid-cell--medium-span-${expanded ? '12' : '4'}`
+          )}
+        >
+          <h2>Top Databases</h2>
```

Comment:
This seems to be all of the databases. Maybe just `Databases`? Or the more verbose `Cross-referenced sequence databases` to try to make it as clear as possible what they are?